alerter.cpp

```diff
@@ -1,31 +1,30 @@
 #include <iostream>
 #include <assert.h>
+#include "alerter_stub.hpp"
 
 int alertFailureCount = 0;
 
-int networkAlertStub(float celcius) {
-    std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+float farenheitToCelcius(float farenheit) {
+    return (farenheit - 32) * 5 / 9;
 }
-
 void alertInCelcius(float farenheit) {
-    float celcius = (farenheit - 32) * 5 / 9;
+    float celcius = farenheitToCelcius(farenheit);

```

Comment:
Test stub needs to be abstracted from production code either by use of interface or function pointer.